### ISSUE TITLE
A naive copy paste of the first instructions of the readme file does not work.

### ISSUE THREAD

**Issue by fmv1992** [just-containers/s6-overlay#412]
Running the examples:

```
# Use your favorite image
FROM ubuntu
ARG S6_OVERLAY_VERSION=3.1.0.0-1
⋯
ENTRYPOINT ["/init"]
```

Followed by the docker commands does not work anymore.

**Comment by GreyXor** [comment]
`ARG S6_OVERLAY_VERSION=3.0.0.2`

**Comment by skarnet** [comment]
Yes, the documentation reflects the current *source*, not the current released binaries. 3.1.0.0 is due soon, but given the high amount of user reports, it's taking some time to get everything addressed. Please be patient.

**Comment by fmv1992** [comment]
Thanks for the your work on this. I'm closing it then.
